functions/Get-DbaRegisteredServerName.ps1

```diff
@@ -57,144 +53,148 @@ function Get-DbaRegisteredServerName {
 			Gets a list of server names in the HR and Accounting groups from the Central Management Server on sqlserver2014a.
 
 		.EXAMPLE
-			Get-DbaRegisteredServerName -SqlInstance sqlserver2014a -Group HR, Accounting -IpAddr
+			Get-DbaRegisteredServerName -SqlInstance sqlserver2014a -Group HR, Accounting -IpAddress
 
 			Gets a list of server IP addresses in the HR and Accounting groups from the Central Management Server on sqlserver2014a.
+
+		.EXAMPLE
+			Get-DbaRegisteredServerName -SqlInstance sqlserver2014a -NoCmsServer
+
+			Gets a list of server names from the Central Management Server on sqlserver2014a, but excludes the cms server name.
+
 	#>
-	[CmdletBinding(DefaultParameterSetName = "Default")]
-	param (
-		[parameter(Mandatory = $true, ValueFromPipeline = $true)]
-		[Alias("ServerInstance", "SqlServer")]
-		[DbaInstanceParameter]$SqlInstance,
-		[PSCredential]$SqlCredential,
+    [CmdletBinding(DefaultParameterSetName = "Default")]
+    param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+		[PSCredential][System.Management.Automation.Credential()]$SqlCredential = [System.Management.Automation.PSCredential]::Empty,
```

Comment:
@jpomfret - it used to be right but we found recently found that it prompts on win7/ps3 so we have to go back to plain ol pscredential :(